src/hooks/useDrawer/useDrawer.test.tsx

```diff
@@ -16,9 +16,7 @@
  * SPDX-License-Identifier: Apache-2.0
  */
 
-import { act } from 'react'
-
-import { renderHook } from '../../test-utils'
+import { act, renderHook } from '../../test-utils'
```

Comment:
Since we are re-exporting the package functions from our module, it seems a good pattern to use our versions so that if in the future we want to provide a different implementation (or enrich the default one) it will take effect throughout the components